Summary.md

```diff
@@ -23,8 +23,12 @@ Design rules can be technical rules, which should be tested automatically and fu
 * <a href="#/core/naming-collections">/core/naming-collections</a>: Use plural nouns to name collection resources.
 * <a href="#/core/changelog">/core/changelog</a>: Publish a changelog for API changes between versions.
 
-#### List of technical rules
+Additional eDelivery compliant rules:  
```

Comment:
Het lijkt me niet wenselijk om deze onderverdeling zo expliciet te maken. Ik zou daarom eerdere pleiten voor een aparte eDelivery module. Design rules die van algemene aard zijn (zoals `/core/backward-compatibility`) kunnen gewoon in het standaard lijstje.